debezium-bom/pom.xml

```diff
@@ -30,6 +30,7 @@
 
         <!-- Tracing -->
         <version.opentelemetry>1.23.0</version.opentelemetry>
+        <version.opentelemetry.testing>2.10.0</version.opentelemetry.testing>
```

Comment:
Had to add the `2.10.0` opentelemetry testing javaagent as the `1.23.0` one would not support JDK 21 and would fail to initialize.

I did it without updating the opentelemetry dependency used in code to avoid introducing other changes.